src/api/payment-requests/payment-requests.md

```diff
@@ -257,6 +257,8 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | expiry            | [Payment Request][] wasn't paid before time out.                  |
 | accept-condition  | A [Payment Condition][] was accepted.                             |
 | decline-condition | A [Payment Condition][] was declined.                             |
+| confirmation      | Funds on a Pre Auth have been drawn down on.                      |
```

Comment:
```suggestion
| confirmation      | Draw down against funds from Pre Auth Authorization.                      |
```

Reply:
This suggestion is not consistent with the tense of the descriptions above. For this reason I am not going to accept this change.